osmtm/views/project.py

```diff
@@ -12,6 +12,7 @@
     TaskState,
     TaskLock,
     License,
+    INVALIDATED,
```

Comment:
I generally prefer not to use the constant directly. I use the class member instead. ie. `Task.state_invalidated`.